classes/PodsI18n.php

```diff
@@ -36,9 +36,7 @@ private function __construct() {
 		self::$instance = $this;
 
 		// Hook all enqueue scripts actions
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
-		add_action( 'login_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+		add_action( 'pods_before_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 74c4e3fb6be84e8c265dfa133b7ee9cf</sub>